code/modules/surgery/healing.dm

```diff
@@ -1,184 +1,315 @@
-/datum/surgery/healing
+/datum/surgery/heal
 	steps = list(/datum/surgery_step/incise,
 				/datum/surgery_step/retract_skin,
 				/datum/surgery_step/incise,
 				/datum/surgery_step/clamp_bleeders,
-				/datum/surgery_step/heal,
+				/datum/surgery_step/heal/four_damages,
 				/datum/surgery_step/close)
 
 	target_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
 	possible_locs = list(BODY_ZONE_CHEST)
-	requires_bodypart_type = 0
+	requires_bodypart_type = FALSE
 	replaced_by = /datum/surgery
-	ignore_clothes = TRUE
 	var/healing_step_type
-	var/antispam = FALSE
 
-/datum/surgery/healing/New(surgery_target, surgery_location, surgery_bodypart)
+/datum/surgery/heal/New(surgery_target, surgery_location, surgery_bodypart)
 	..()
 	if(healing_step_type)
 		steps = list(/datum/surgery_step/incise/nobleed,
 					healing_step_type, //hehe cheeky
 					/datum/surgery_step/close)
 
+/datum/surgery/heal/four_damages
+	ignore_clothes = TRUE
+	var/antispam = FALSE
+
+
 /datum/surgery_step/heal
 	name = "repair body"
 	implements = list(/obj/item/hemostat = 100, TOOL_SCREWDRIVER = 65, /obj/item/pen = 55)
 	repeatable = TRUE
 	time = 25
+
+/datum/surgery_step/heal/four_damages/
```

Comment:
Why exactly a new subtype? Is there another type branching off /datum/surgery_step/heal? Aren't you just making your life harder here?

Reply:
yes/no.

I'm planning on using the heal step for organ damage which will function differently.

If I don't get to it I can easily f/r this back.